client/actions/checks.js

```diff
@@ -20,12 +20,20 @@ function deleteCheck(status, payload = null) {
   }
 }
 
+function constructError(check, error) {
+  return {
+    ...error,
+    message: error.message,
+    checkId: check.checkId
+  }
+}
+
```

Comment:
Another option would be to use a custom error class, ES6 style. What do you think? 😄